.env.example

```
@@ -0,0 +1,4 @@
+NODE_ENV='development'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)